File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06IO1/Command/Command/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    internal class Sportsmen
    {
        //private string _name;
        //public string name { get { return _name; } private set { _name = value; } }
        public string name { get; private set; }
        public Sportsmen(string name) { this.name = name; }
        public string Run() { return "Running"; }
        public string Jump() { return "Jumping"; }
        public string Swim() { return "Swiming"; }
        public string execute(SportsmenCommand command) { return name+": "+command.execute(this); }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2023w/ID07IO1/CommandPattern/src/Sportsmen.java
public class Sportsmen
{
   private final String name;

   public Sportsmen(String name) { this.name=name; }
   public String Run() { return "Running"; }
   public String Jump() { return "Jumping"; }
   public String Swim() { return "Swiming"; }
   @Override public String toString() { return name; }
}
